openff/toolkit/_tests/test_toolkits.py

```diff
@@ -193,8 +193,6 @@ def get_mini_drug_bank(toolkit_class, xfail_mols=None):
     "DrugBank_1962",
     "DrugBank_5043",
     "DrugBank_2519",
-    "DrugBank_7124",
-    "DrugBank_6865",
```

Comment:
I can't find the relevant change in the RDKit changelog, but this basically reverts part of https://github.com/openforcefield/openff-toolkit/pull/895